benchmarks/nvector/serial/test_nvector_performance_serial.c

```diff
@@ -255,6 +255,7 @@ void ClearCache()
     sum = SUN_RCONST(0.0);
     for (i=0; i<N; i++)
       sum += data[i];
+    (void) sum;
```

Comment:
A guess this prevents a warning, but I would expect the compiler to optimize this entire function into nothing. `ClearCache` is seemingly never called here or in most benchmarks.

Reply:
Its here precisely to get rid of the warning.